KnockKnock-iOS/Repository/SNS/AccountManager.swift

```diff
@@ -7,10 +7,9 @@
 
 import Foundation
 
-import KakaoSDKUser
-
-protocol SocialLoginManagerProtocol {
-  func requestToken(
+protocol AccountManagerProtocol {
+  func logIn(
```

Comment:
```suggestion
  func signIn(
```
이렇게 네이밍을 하는것이 더 좋은 방법이겠습니다!